cmds/which/which_other.go

```diff
@@ -0,0 +1,10 @@
+// +build !linux
+// Copyright 2016-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+func canExecute(path string) bool {
+	return true
```

Comment:
actually the first order approx in the linux one would be much better here.

Reply:
I'll make _linux into _unix.

Reply:
The problem with lookpath is that its test for executability is wrong (I tested it) and we'd be better off with NOT using it :-(
maybe clone it ... I would say let's not change which for now, but fix it better later with fxstatat?